arci-urdf-viz/src/client.rs

```diff
@@ -84,6 +103,77 @@ impl UrdfVizWebClient {
             send_robot_origin(&base_url, pose).unwrap();
         });
     }
+    pub fn run_send_joint_positions_thread(&mut self) {
+        if self.send_joint_positions_thread.is_some() {
+            panic!("send_joint_positions_thread is running.");
+        }
```

Comment:
If we can make this private, and automatically runs in the cunstructor, we can remove this check and avoid `panic!`. Is it possible?

Reply:
Or use scoped thread (crossbeam)?